tests/unittests/odm_tests.py

```diff
@@ -57,3 +57,51 @@ def test_doc_copy_with_new_id(self):
 
         finally:
             dataset_doc.delete()
+
+
+class GetIndexedValuesTests(unittest.TestCase):
+    def test_get_indexed_values(self):
+        try:
+            dataset = fo.Dataset()
+
+            samples = [
+                fo.Sample(filepath="sample_%d.jpg" % i) for i in range(10)
+            ]
+            new_ids = dataset.add_samples(samples)
+
+            dataset.add_sample_field("new_field", fo.IntField)
+            dataset.set_values("new_field", [i for i in range(10)])
+            dataset.create_index("new_field")
```

Comment:
`create_index` is a blocking call so `new_field` index should exist, but maybe we can add a check in unit test to make sure ~

Reply:
yeah it's blocking. Failing the check would would be the same as failing the test though? I think we can add the check if it starts flaking, but for now, since it doesn't seem to be problematic, no reason to overcomplicate it